tests/phpunit/src/Application/ExceptionApplicationTest.php

```diff
@@ -18,26 +18,49 @@
  */
 class ExceptionApplicationTest extends TestBase {
 
+  protected $kernel;
+
   public function setUp($output = NULL):void {
     parent::setUp($output);
     // If kernel is cached from a previous run, it doesn't get detected in code
     // coverage reports.
     $this->fs->remove('var/cache');
+    $this->kernel = new Kernel('dev', 0);
```

Comment:
Why moving the kernel initialization into `setUp` while it used only for only one test before? Anyway I'm not super opinionated, was just wondering...

Reply:
I'm not sure I understand. The Kernel needs to boot for every test case (since it holds the application container), so `setUp` seems the natural place to initialize it.